pages/linux/xsel.md

```diff
@@ -0,0 +1,16 @@
+# xsel
+
+> Selection manipulation tool.
+> For example xsel can copy STDIN to clipboard or print clipboard to STDOUT.
+
+- Copy output to clipboard:
+
+`echo 123 | xclip -bi`
+
+- Paste clipboard:
```

Comment:
Why not "Paste clipboard into a file"? We could have a separate example where it is piped into another command, if you want to convey the generality of this operation.